src/Corax/Queries/TermMatch.cs

```diff
@@ -130,27 +136,46 @@ static int AndWithFunc(ref TermMatch term, Span<long> buffer, int matches)
                         return 1;
                     }
                 }
+
                 return 0;
             }
 
+            [MethodImpl(MethodImplOptions.AggressiveInlining)]
+            static void ScoreFunc(ref TermMatch term, Span<long> matches, Span<float> scores, float boostFactor)
+            {
+                using (term._bm25Relevance)
+                    term._bm25Relevance.Score(matches, scores, boostFactor);
+            }
```

Comment:
Is calling dispose the right thing to do here?

Reply:
bump

Reply:
Calling score() is a one-time operation during sorting, so I believe we can do it. On the other hand, is it worth it, since it will be disposed of immediately after query execution?